src/pages/check-email/_icon-grid.js

```diff
@@ -45,12 +45,24 @@ const StyledText = styled(Text)`
 `
 
 const Col = ({ Icon, alt, content }) => (
-    <GridCol>
-        <img src={Icon} alt={alt} />
-        <StyledText color="black-3" lh="1.55" mt="0.8rem">
-            {content}
-        </StyledText>
-    </GridCol>
+    <div>
+        <Show.Desktop>
+            <GridCol>
+                <img src={Icon} alt={alt} />
+                <StyledText color="black-3" lh="1.55" mt="0.8rem">
+                    {content}
+                </StyledText>
+            </GridCol>
+        </Show.Desktop>
+        <Show.Mobile>
+            <GridCol>
+                <img src={Icon} alt={alt} width="40px" height="40px" />
```

Comment:
since the change is only on img tag, you can use @media ${device.tabletL} {...your styles}

you can extend img to styled component 
```
const Img = styled.img`
@media ...

your styles
`
```

Reply:
Thanks for pointing this out.